spec/mailers/track_mailer_spec.rb

```diff
@@ -28,7 +28,7 @@
                            :url_name => 'test-name',
                            :get_locale => 'en',
                            :should_be_emailed? => true)
-        klass = ActiveRecord::Relation::ActiveRecord_Relation_User
+        klass = ActiveRecord::Relation
```

Comment:
um, I suspect the **correct** version may be that it's a `User::ActiveRecord_Relation` now rather than an `ActiveRecord::Relation::ActiveRecord_Relation_User`